### ISSUE TITLE
TypeError: addtext() takes exactly 3 arguments

### ISSUE THREAD

**Issue by jgphpc** [hpcugent/mympingpong#20]
Hi,

* ./mympingponganalysis.py -f ./eff.h5

```
Traceback (most recent call last):
  File "./mympingponganalysis.py", line 310, in <module>
    ppa.plot(go.options.colormap, go.options.input, 
                   go.options.show, go.options.save, lscale, lmask)
  File "./mympingponganalysis.py", line 259, in plot
    self.addtext(self.meta, plt.subplot(gs1[0:1, 5:10]), fig1)
TypeError: addtext() takes exactly 3 arguments (4 given)
```

Quick quickaround is to comment out https://github.com/hpcugent/mympingpong/blob/master/bin/mympingponganalysis.py#L259

Same issue with matplotlib/1.3.1 or matplotlib/2.0.2

**Comment by boegel** [comment]
@jgphpc duplicate of #18?

**Comment by jgphpc** [comment]
Yes you're right. I guess we can close both now.

**Comment by boegel** [comment]
fixed in #21
